plugins/org.jkiss.dbeaver.ext.athena/src/org/jkiss/dbeaver/ext/athena/model/AthenaSQLDialect.java

```diff
@@ -25,4 +25,11 @@ public class AthenaSQLDialect extends GenericSQLDialect {
     public AthenaSQLDialect() {
         super("Athena", "aws_athena");
     }
+
+    // https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html#tables-databases-columns-names-complex-types
+    @Override
+    public boolean validIdentifierPart(char c, boolean quoted) {
+        return (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || Character.isDigit(c) || c == '_' || (quoted
+            && validCharacters.indexOf(c) != -1);
```

Comment:
You probably want this, don't you?
```suggestion
        return SQLUtils.isLatinLetter(c) || Character.isDigit(c) || c == '_' || (quoted && validCharacters.indexOf(c) != -1);
```